scaffold/tugboat/tugboat.acquia-example.yml

```diff
@@ -0,0 +1,106 @@
+        set -eux
+        echo "Initializing..."
+        # Install prerequisite packages
+        apt-get update
+        apt-get install -y mariadb-client libldap2-dev
+
+        # Install drush-launcher
+        wget -O /usr/local/bin/drush https://github.com/drush-ops/drush-launcher/releases/download/0.10.1/drush.phar
+        chmod +x /usr/local/bin/drush
+
+        # Link the document root to the expected path. Tugboat uses /docroot
+        # by default. So, if Drupal is located at any other path in your git
+        # repository, change that here. This example links /web to the docroot
+        [ -d "${DOCROOT}" ] || ln -snf "${TUGBOAT_ROOT}/web" "${DOCROOT}"
+
+        docker-php-ext-install opcache
+        a2enmod headers rewrite
+
+      # Commands that import files, databases,  or other assets. When an
+      # existing preview is refreshed, the build workflow starts here,
+      # skipping the init step, because the results of that step will
+      # already be present.
+      update: |
+        set -eux
+        echo "Updating..."
+
+        # We intentionally do not set up syncing of static assets, as we have
+        # decided to always use Stage File Proxy:
+        # https://architecture.lullabot.com/adr/20210729-stage-file-proxy/
+
+        # Set up permissions for the files directories.
+        chgrp -R www-data "${DOCROOT}/sites/default/files"
+        chmod -R g+w "${DOCROOT}/sites/default/files"
+        chmod 2775 "${DOCROOT}/sites/default/files"
+
+        composer install
+
+        # Install Drupal using the standard profile.
+        vendor/bin/task drupal:install
+
+        # To use an existing site, add the appropriate environment variables
+        # as document at <LINK> and instead run:
+        # vendor/bin/task <provider>:fetch-db
+        # vendor/bin/task drupal:import-db
+
+      # Commands that build the site. This is where you would add things
+      # like feature reverts or any other drush commands required to
+      # set up or configure the site. When a preview is built from a
+      # base preview, the build workflow starts here, skipping the init
+      # and update steps, because the results of those are inherited
+      # from the base preview.
+      build: |
+        set -eux
+        vendor/bin/task drupal:update
+        vendor/bin/drush user:login
+
+  # What to call the service hosting MySQL. This name also acts as the
+  # hostname to access the service by from the php service.
+  mariadb:
+
+    image: tugboatqa/mariadb:10
+
+    # A set of commands to run while building this service
+    commands:
+
+      # Commands that import files, databases,  or other assets. When an
+      # existing preview is refreshed, the build workflow starts here,
+      # skipping the init step, because the results of that step will
+      # already be present.
+      update:
+        - echo "Nothing to do as we import databases from the Drupal service."
+
+  memcached:
+
+    image: tugboatqa/memcached:1
```

Comment:
Wow, we literally only need two lines here? I figured there would be a tiny bit more.

Seeing this, part of me thinks we could instead combine multiple files together, so we don't have to duplicate the first 100 lines. However, I'm not sure that we won't need _something_ custom for environment variables to do something like downsync Acquia databases into base previews.

Reply:
_Seeing this, part of me thinks we could instead combine multiple files together, so we don't have to duplicate the first 100 lines. However, I'm not sure that we won't need something custom for environment variables to do something like downsync Acquia databases into base previews._

I don't understand your comment, could you please explain a little more. Thank you